build/parents/public/pom.xml

```diff
@@ -231,7 +230,12 @@
                                 </moduleInfo>
                             </module>
                             <jdepsExtraArgs>
-                                <arg>--multi-release=9</arg>
+                                <!--
+                                    Running jdpes tool will fail without this option, if there are any multi-release jars
+                                    on the path. And since some modules might have such jars, e.g. util-common has Jandex,
+                                    it's safer to add this additional option:
```

Comment:
Okay. Does this mean we generate a multi-release JAR, or just that we pass an extra argument to `jdeps` during the build, with no impact on the produced JAR?

Reply:
From what it looks: only an extra argument. I've looked inside the resulting jar - module-info is just in the root, and there's no additional versions folder for multi-release